code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -36,6 +36,13 @@
 /mob/living/carbon/human/promethean/Initialize(var/new_loc)
 	return ..(new_loc, SPECIES_PROMETHEAN)
 
+/mob/living/carbon/human/zaddat/New(var/new_loc)
+	..(new_loc, SPECIES_ZADDAT)
+
+/mob/living/carbon/human/machine/New(var/new_loc)
```

Comment:
I'm pretty sure this was deliberately removed on the master, can you please double check that your code is up to date?